commit c1379a077079f1b1f9d2837a9524dd29ea7a88fa (refs/changes/07/207/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-08-04T15:13:11-07:00 (3 years, 2 months ago)
    
    TOOLS-1508 prep_dataset_in_jpc.sh using wrong "mantapath" dir for built image export

diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index afc1332..3e79b09 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -362,7 +362,7 @@ if [[ -n ${SDC_LOCAL_BUILD} ]]; then
     curl -sS -f -o ${output_dir}/${image_filename} ${SDC_IMGAPI_URL}/images/${image_id}/file
 
 else
-    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | cut -d '-' -f1,2)/${build_name}
+    mantapath=/${SDC_ACCOUNT}${MG_OUT_PATH}/${build_name}/$(echo ${image_version} | sed -E 's/-g[a-f0-9]{7}$//')/${build_name}
     mmkdir -p ${mantapath}
 
     manta_bits=/tmp/manta-exported-image.$$
